torch/quantization/adaround_fake_quantize.py

```diff
@@ -0,0 +1,102 @@
+import torch
+import torch.nn.init as init
+import math
+from torch.quantization.fake_quantize import FakeQuantize
+from torch.quantization.observer import MovingAverageMinMaxObserver
+from torch.quantization.qconfig import *
+from torch.quantization.fake_quantize import *
+
+
+class AdaRoundFakeQuantize(FakeQuantize):
+    ''' This class is an extension of the FakeQuantize module and is used similarly.
+    The difference between the two is that you can use the continous_V attribute to alter the
+    default rounding scheme during quantization. The defualt is to round to the nearest number,
+    but applying an optimizer on this attribute can yield a better rounding scheme and improve
+    quantization results.
+
+    '''
+    adaround_default_attribute_settings = {'continous_V': None,
+                                           'beta_high': 8,
+                                           'beta_low': 2,
+                                           'norm_scaling': 10,
+                                           'regularization_scaling': .1}
+
+    def __init__(self, *args, **keywords):
```

Comment:
I feel it might be better to just spell out the arguments in the list, also please add a comment for all the arguments